models/org_team.go

```diff
@@ -444,6 +478,14 @@ func UpdateTeam(t *Team, authChanged bool) (err error) {
 		}
 	}
 
+	// Add all repositories to the team if it has access to all of them.
+	if t.IncludesAllRepositories {
+		err = t.addAllRepositories(sess)
+		if err != nil {
+			return fmt.Errorf("addAllRepositories: %v", err)
+		}
+	}
+
```

Comment:
else? If some team has `IncludesAllRepositories` true and then change to false?

Reply:
The team will keep access to the currently existing repositories, new repositories won't be added anymore.
The user will be able to add or remove repositories as he wants.

Do you think we should remove all repositories from the team?

Reply:
Oh. Batch remove repositories from team should be another PR.